mlir/utils/performance/tuningRunner.py

```diff
@@ -46,10 +49,8 @@ def verifyModeFlags(verifyMode: str) -> str:
 
 #Run a gemm or conv config and verify it
 def verifyKernelWithPerfConfig(perfConfig, config, paths: Paths, options: Options) -> float:
-    print(f"Verifying with perfConfig = {perfConfig}")
+    print(f"Verifying with perfConfig = {perfConfig}", file=sys.stderr)
```

Comment:
Changed all the running-commentary prints to print to stderr, so I could optionally send the results to stdout.